package.json

```diff
@@ -12,10 +12,12 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "^15.1.0",
+    "@angular/cdk": "^15.1.2",
```

Comment:
what is this for?

Reply:
i dont know I think it was added along with angular materials